i_have_a_database.html

```diff
@@ -0,0 +1,140 @@
+            <li><a href="https://github.com/ga4gh/mme-apis" target="_blank"><span class='glyphicon glyphicon-new-window pull-right text-muted'></span> API GitHub Repository</a></li>
+            <li><a href="steering_committee.html"><span class='glyphicon glyphicon-save pull-right text-muted'></span> Steering Committee</a></li>
+          </ul>
+        </li>
+        <li class='col-sm-3 padding-left-none padding-right-none'> <a href="/#Supporters" class='text-center  text-uppercase text-white text-md'><span class='glyphicon glyphicon-star'></span> <span class='visible-xs-inline visible-lg-inline'>Exchange </span>Participants </a></li>
+        <li class='col-sm-1 padding-left-none padding-right-none'><a href="mailto:dazzariti@partners.org" class='text-center text-uppercase text-white text-md'><span class='glyphicon glyphicon-phone-alt'></span></a></li>
+      </ul>
+    </div>
+    <!-- /.navbar-collapse --> 
+  </div>
+  <!-- /.container-fluid --> 
+</nav>
+<div class='container-fluid  bg-grey'>
+<div class='container'>
+  <div class='row-fluid clearfix padding-bottom-lg padding-top-sm'>
+    <div class='col-sm-12 col-xs-12'>
+    <a id="top" name="top"></a>
+      <h1 class='text-white'>Connecting your genomic matchmaking service through the Matchmaker Exchange API</h1>
+      <h4 class='text-white'>Learn more about the API here on the MME GitHub Repository: <a class="text-white" href="https://github.com/ga4gh/mme-apis" target='_blank'>https://github.com/ga4gh/mme-apis</a></h4>
+      <h4 class='text-white'>If you are interested in implementing the API, please contact us at <a class="text-white" href="mailto:api@matchmakerexchange.org">api@matchmakerexchange.org</a> to get started.  We can walk you through the requirements listed below. </h4>
+      <div class="row"></div>
+    </div>
+  </div>
+</div> 
+</div>
+<div class='container'>
+  <div class='row-fluid clearfix padding-bottom-lg padding-top-lg'>
+    <div class='col-sm-12 col-xs-12'>
+      <p>To become a Matchmaker Exchange Service, each new site must achieve the following:</p>
+      <ol>
+        <li>
+          <p>Require users to deposit case data to undertake a federated query across the MME service providers</p>
+        </li>
+        <li>
+          <p>Establish a minimum of two point-to-point API connections to other MME services</p>
+        </li>
+        <li>
+          <p>Contain content that is considered by the MME steering committee to be useful for matching, including the flagging of, or ability to prioritize, candidate genes</p>
+        </li>
+        <li>
+          <p>Successfully implement matching algorithms using test data</p>
+        </li>
+        <li>
+          <p>During user queries, enable dual notification of data requester (i.e. the querier) and prior data depositor (i.e. the queried) including sharing the identities and contact information for each</p>
```

Comment:
I think we may need to review the wording of this requirement. Currently within Decipher, the data depositor is only notified if the querier decides to follow up on the request (As in, chooses to make contact). In addition, Decipher does not make the identity of the data depositor known. It is up to the original depositor of Decipher data to decide if they wish to responde to any contact request. This is the same way we handle contact requests made by unauthenticated user looking at public consented data on Decipher.